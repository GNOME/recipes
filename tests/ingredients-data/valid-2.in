1/2	pound	butter	
2.5		egg	
15		salt	
