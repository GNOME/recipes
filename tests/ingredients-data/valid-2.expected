AMOUNT 0.500000
UNIT lb
NAME Butter

AMOUNT 2.500000
UNIT 
NAME Egg

AMOUNT 15.000000
UNIT 
NAME Salt

